Makefile

```diff
@@ -1,7 +1,43 @@
+ROOT:=$(shell git rev-parse --show-toplevel)
+COMMIT:=$(shell git rev-parse --short HEAD)
+MIN_RUNS:=25
+
 src/scanners.rs: src/scanners.re
 	re2rust -W -Werror -i --no-generation-date -o $@ $<
 	cargo fmt
 
 bench:
 	cargo build --release
 	(cd vendor/cmark-gfm/; make bench PROG=../../target/release/comrak)
+
+binaries: build-comrak-branch build-comrak-master build-cmark-gfm build-pulldown-cmark
+
+build-comrak-branch:
+	cargo build --release
+	cp ${ROOT}/target/release/comrak ${ROOT}/benches/comrak-${COMMIT}
+
+build-comrak-master:
+	git clone https://github.com/kivikakk/comrak.git --depth 1 --single-branch ${ROOT}/vendor/comrak || true
+	cd ${ROOT}/vendor/comrak && \
+	cargo build --release && \
+	cp ./target/release/comrak ${ROOT}/benches/comrak-main
+
+build-cmark-gfm:
+	cd ${ROOT}/vendor/cmark-gfm && \
+	make && \
+	cp build/src/cmark-gfm ${ROOT}/benches/cmark-gfm
+
+build-pulldown-cmark:
+	cd ${ROOT}/vendor/pulldown-cmark && \
+	cargo build --release && \
+	cp target/release/pulldown-cmark ${ROOT}/benches/pulldown-cmark
+
+bench-comrak: build-comrak-branch
```

Comment:
wdyt about adding [markdown-it.rs](https://github.com/rlidwka/markdown-it.rs) to the mix? It would be nice to have [markdown-rs](https://github.com/wooorm/markdown-rs), but as you said, at the moment there is not a command line version.

One thing we have to remember is that this really isn't testing exactly apples to apples. For example `markdown-rs` doesn't support footnotes and task lists directly yet. Though some [plugins](https://github.com/chrisjsewell/markdown-it-plugins.rs) have just been added for that.

A follow-on PR might have a small harness program for the Rust implementations, which would link in all the different `lib` versions.  The harness would take a switch, and then run the specified implementation, with the proper switches set to make them all use similar functionality.

Reply:
Sure, I didn't know about it ; so will take a look and try adding it as well.